tests/test_vector3.py

```
@@ -15,166 +15,694 @@ def runTest(self):
 
         np.random.seed(2599)
 
-        # arrays of wrong shape raise ValueError
-        self.assertRaises(ValueError, Vector3, np.random.randn(3,4,5))
+        # Test basic construction
+        v1 = Vector3([1., 2., 3.])
+        self.assertEqual(v1.shape, ())
```

Ruff (0.14.7)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)